MetaMask/metamask-extension
# Open sea security provider warning message

## Explanation
* Fixes #16258

## Manual Testing Steps
1. Go to [`test-dapp`](https://metamask.github.io/test-dapp/)
2. Select one of the options:
- `SEND LEGACY TRANSACTION`
- `SEND EIP 1559 TRANSACTION`
- `ETH SIGN`
- `PERSONAL SIGN`
- `SIGN TYPED DATA`
- `SIGN TYPED DATA V3`
- `SIGN TYPED DATA V4`
- `SIGN IN WITH ETHEREUM`
- `SIGN IN WITH ETHEREUM (W/ RESOURCES)`
- `SIGN IN WITH ETHEREUM (BAD ACCOUNT)`
- `SIGN IN WITH ETHEREUM (MALFORMED)`

The warning message will appear depending on the input data sent to the `OpenSea` security provider.
